tests/test_soup_searcher.py

```diff
@@ -0,0 +1,29 @@
+import pytest
```

Comment:
random note, the convention for this folder is to be named `test` instead of `tests`....although it does have tests.

This is also common in multiple languages